de.metas.util/src/main/java/de/metas/util/StringUtils.java

```diff
@@ -993,4 +993,12 @@ else if (capitalize)
 		}
 		return new String(data);
 	}	// initCap
+
+	/**
+	 * @param in	input {@link String}
+	 * @return {@param in} if != null, empty string otherwise
+	 */
+	public static String nonNullOrEmpty(final String in) {
```

Comment:
we already have `de.metas.util.Check.isEmpty(String, boolean)` which can also handle whitespace strings. please use it instead.

Reply:
They are not doing the same thing; I need an method that will return an empty String if the input String is null

Reply:
I think I should find a better name for the method since this one seems misleading.